### DISCUSSION TITLE
[Timeseries] Add option to set fixed width in Legend

### DISCUSSION THREAD

**Discussion by simPod** [grafana/grafana#48561] [Dashboards, Panels & Visualizations]
_I was told to re-open the bug report as FR by @kirederik [here](https://github.com/grafana/grafana/issues/46653#issuecomment-1109561776)_

The Graph panel had option to fix legend width. That resulted in nicely aligned panels where series area had the same width. So it was actually possible to compare two panels that were below each other with a quick glance. 

As @Karsonito showcased, it's unusable in new Timeseries panel as it lacks fixed legend width support. (shared crosshair is not a solution nor workaround)

![image](https://user-images.githubusercontent.com/327717/165518237-35c3bdd5-dd53-49ca-8722-d5ad727a2455.png)

**Comment by mellieA** [comment]
@simPod feature requests have been migrated to discussions in the grafana repo. I've moved this issue into discussions for you.

**Comment by malaiam** [comment]
Guys, thanks for the hard work you're putting into this. Grafana is a really nice presentation for our data!

I want to bring some focus on this, as this is not a "discussion", this is a feature which used to work and now it's been left behind.
We have plenty of dashboards where the fixed width of the legends is important so we can easily identify and confirm patterns in multiple graphs. Also the shared crosshair is not working anymore, so that cannot be used as an alternative either.

Things like this matter when you work in operations and need to identify root causes quickly, so please add the feature back to the graph. Otherwise, at least from where I'm looking, I don't see the benefit of switching over to the new timeseries graph. 

@kirederik FYI ☝️

**Comment by nmarrs** [comment]
This has been addressed by https://github.com/grafana/grafana/pull/49126
